code/modules/unit_tests/atmos/test_ventcrawl.dm

```diff
@@ -19,7 +24,10 @@
 	Fail("Couldn't find spawned test object of type: [test_object_type].")
 
 /datum/unit_test/ventcrawl/Run()
-	setup_test_area()
+	var/datum/milla_safe/ventcrawl_test_setup/milla = new()
+	milla.invoke_async(src)
+	while(!setup_complete)
+		sleep(0.01)
```

Comment:
```suggestion
		sleep(world.tick_lag)
```
May as well sleep till next tick

Reply:
Yeah, worst case is we lose 1/20th of a second in a unit test. Oh, horrors.